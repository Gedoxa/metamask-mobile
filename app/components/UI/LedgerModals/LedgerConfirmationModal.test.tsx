import { shallow } from 'enzyme';
import { renderScreen } from '../../../util/test/renderWithProvider';
import LedgerConfirmationModal from './LedgerConfirmationModal';


describe('LedgerConfirmationModal', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <LedgerConfirmationModal onConfirmation={jest.fn()} onRejection={jest.fn()} deviceId={'test'} />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
