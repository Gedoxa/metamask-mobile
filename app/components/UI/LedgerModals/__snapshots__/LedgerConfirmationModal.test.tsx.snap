// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LedgerConfirmationModal should render correctly 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderTopLeftRadius": 10,
      "borderTopRightRadius": 10,
      "height": 450,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "marginTop": 35,
      }
    }
  >
    <Component />
  </View>
</RCTSafeAreaView>
`;
